src/components/InfoViewer/InfoViewer.tsx

```diff
@@ -26,28 +27,35 @@ const InfoViewer = ({
     size,
     className,
     multilineLabels,
-}: InfoViewerProps) => (
-    <div className={b({size}, className)}>
-        {title && <div className={b('title')}>{title}</div>}
-        {info && info.length > 0 ? (
-            <div className={b('items')}>
-                {info.map((data, infoIndex) => (
-                    <div className={b('row')} key={data.label + infoIndex}>
-                        <div className={b('label')}>
-                            <div className={b('label-text', {multiline: multilineLabels})}>
-                                {data.label}
+    renderOnEmptyState,
```

Comment:
`renderOnEmptyState = () => null` not to provide this prop if you just want to render null (it seems it's a common case).

Reply:
no, the common case is to render `No ${title} data`